packages/terra-functional-testing/src/commands/axe/run.js

```diff
@@ -20,8 +20,19 @@ const runAxe = (overrides = {}) => {
     injectAxe(axeOptions);
   }
 
+  /**
+    * This rule was introduced in axe-core v3.3 and causes failures in many Terra components.
+    * The solution to address this failure vary by component. It is being disabled until a solution is identified in the future.
+    *
+    * Reference: https://github.com/cerner/terra-framework/issues/991
+    */
+  const ruleOverrides = {
+    'scrollable-region-focusable': { enabled: false },
+  };
+
   // Merge the global rules and overrides together.
   const rules = {
+    ...ruleOverrides,
     ...axeOptions && axeOptions.rules.reduce((acc, rule) => ({ ...acc, [rule.id]: rule }), {}),
     ...overrides.rules,
```

Comment:
Looks like our [current implementation](https://github.com/cerner/terra-toolkit-boneyard/blob/385d3210d6045f44246fd9d8503dc0f7d6f003d3/src/wdio/services/TerraCommands/determine-test-options.js#L52) also honors `options.axeRules` if there is no `options.rules`. I can't find what the difference is between these `axeRules` and `rule`. Do we need to check for `axeRules` here?

Reply:
I believe that's left over from an earlier assertion implementation.  I'm voting to remove it. We shouldn't have two identical options keys.

I started a list of the breaking changes here I plan to PR after some some adjustments to get the migration guide started:

https://github.com/cerner/terra-toolkit/compare/migration-guide-v1